src/datasource/components/QueryEditor.tsx

```diff
@@ -395,7 +395,8 @@ export class QueryEditor extends PureComponent<Props, State> {
     if (!this.props.datasource.grafanaLiveEnabled) {
       return null;
     }
-
+    // Fixing weird typing issue
+    const FVM = FieldValidationMessage as any;
```

Comment:
This one is weird 😬 --in 9x it is ok, but the 8.3x part has weird inherited types that do not expect childe attributes.